polars/dataframe.ts

```diff
@@ -283,13 +282,13 @@ export interface DataFrame
    * @param name
    * @example
    * ```
-   * >  df = pl.DataFrame({
+   * >  const df = pl.DataFrame({
    * ...    "foo": [1, 2, 3],
    * ...    "bar": [6.0, 7.0, 8.0],
    * ...    "ham": ['a', 'b', 'c'],
    * ...    "apple": ['a', 'b', 'c']
-   * ...  })
-   * >  df.drop(['ham', 'apple'])
+   * ...  });
+   * >  console.log(df.drop(['ham', 'apple']).toString());
```

Comment:
you shouldn't need `toString()` here. the `node.util.inspect` already properly displays the dataframe in `console.log`

Reply:
but It does not work in `Bun` b/c I get `ProxyObject` without `.toString()`:
```
ProxyObject {
  foo: shape: (3,)
Series: 'foo' [f64]
[
	1.0
	2.0
	3.0
],
  bar: shape: (3,)
Series: 'bar' [f64]
[
	6.0
	7.0
	8.0
],
}
```

Reply:
if that's the case, that'd be a bug in bun. The toString should be unnecessary.

Reply:
I [pinged](https://github.com/oven-sh/bun/issues/4121) Jarred about this issue. Thx

Reply:
I am not sure if it's a bun issues, b/c this works fine in bun:
```
console.log(
   pl.DataFrame([{ a: 1, b: 2,}])[Symbol.for("nodejs.util.inspect.custom")]()
);
```

It results in:
```
shape: (1, 2)
┌─────┬─────┐
│ a   ┆ b   │
│ --- ┆ --- │
│ f64 ┆ f64 │
╞═════╪═════╡
│ 1.0 ┆ 2.0 │
└─────┴─────┘
```

Reply:
yeah this is definitely an issue with bun. If they claim to be 100% interopable with node.js, then the outputs should be the same. I'd suggest opening an issue with them.

Reply:
yes the symbol works as expected, but console.logging without it does not. 

```js
console.log(pl.DataFrame({a: [1]}))
```

this should be identical in both. If it's not, then that is a bug in bun.

Reply:
[Checking](https://github.com/oven-sh/bun/pull/4243#issuecomment-1949106097) with the Bun team.